### DISCUSSION TITLE
Connect when the screen is locked.

### DISCUSSION THREAD

**Discussion by kimiwanano** [rustdesk/rustdesk#6541] [General]
The current rustdesk seems to be unable to implement remote connection when the screen is locked? For example, when host A connects to host B, if host B is locked, the connection password for the rustdesk service on host B cannot be seen, and the authorization information cannot be received. At this time, remote connection cannot be achieved.

**Comment by automate-this** [comment]
Still the case in 2025. Set permanent password, approve-mode = click, allow-logon-screen-password = true.

allow-logon-screen-password should be working for logged out users and locked users, but only works for logged out right now.

Please implement this. This is basic functionality.

**Comment by xlionjuan** [comment]
I never get the problem about this on Linux, Windows and macOS

**Reply by automate-this** [reply]
i guess you have approve-mode = password or approve-mode = password-click. Then it would work.

**Reply by xlionjuan** [reply]
I just set the password, and it just works.

**Reply by automate-this** [reply]
then approve-mode = password-click is set, which is the default. 
but when only "click" is set, it does not work anymore even though allow-logon-screen-password is set to true
